fiks-meldingsformidler/src/main/java/no/difi/meldingsutveksling/ks/mapping/ForsendelseMapper.java

```diff
@@ -26,15 +36,65 @@
 import java.util.GregorianCalendar;
 import java.util.List;
 import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static com.google.common.base.Strings.isNullOrEmpty;
 
 @Slf4j
 public class ForsendelseMapper {
+
     private IntegrasjonspunktProperties properties;
     private ServiceRegistryLookup serviceRegistry;
+    private MessagePersister persister;
 
-    public ForsendelseMapper(IntegrasjonspunktProperties properties, ServiceRegistryLookup serviceRegistry) {
+    public ForsendelseMapper(IntegrasjonspunktProperties properties,
+                             ServiceRegistryLookup serviceRegistry,
+                             MessagePersister persister) {
         this.properties = properties;
         this.serviceRegistry = serviceRegistry;
+        this.persister = persister;
+    }
+
+    public SendForsendelseMedId mapFrom(ConversationResource cr, X509Certificate certificate) {
+        Forsendelse.Builder<Void> forsendelse = Forsendelse.builder();
+        forsendelse.withEksternref(cr.getConversationId());
+        forsendelse.withKunDigitalLevering(false);
+        forsendelse.withSvarPaForsendelse(""); // FIXME: receiver.ref
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1134)